### PR TITLE
aurora-cli update (0.19.0, high-sierra)

### PR DESCRIPTION
**Pull request by thinker0**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----
https://github.com/Homebrew/homebrew-core/issues/18493

### CODE REVIEW

**Comment by ilovezfs** [comment]
What is the status of the patch upstream? It seems like they think 10.13 has been fixed but we're still having to patch.

**Comment by thinker0** [comment]
@ilovezfs 

https://reviews.apache.org/r/44607/ in progress

**Comment by ilovezfs** [comment]
@thinker0 I'm confused. That says
```
Review Request #44607 - Created March 10, 2016 and submitted 1 year, 8 months ago 
```
and looks unrelated.

**Comment by thinker0** [comment]
@ilovezfs Why can not we proceed?

 https://reviews.apache.org/r/63750/

File: Formula/aurora-cli.rb [hunk 1]
```diff
@@ -11,9 +11,15 @@ class AuroraCli < Formula
     sha256 "62e565f0d65601f88b11d8c5f5bfad89d2f1661144072876fc705e50b4284329" => :yosemite
   end
 
+  patch do
```
Comment (ilovezfs) [inline Formula/aurora-cli.rb]: This should be inside a `if MacOS.version == :sierra ... end`

File: Formula/aurora-cli.rb [hunk 2]
```diff
@@ -11,9 +11,15 @@ class AuroraCli < Formula
...
+    url "https://raw.githubusercontent.com/thinker0/aurora/pants-fix-high-sierra/pants_version_1.4.0_dev20.diff"
```
Comment (ilovezfs) [inline Formula/aurora-cli.rb]: This will need to go in a PR to https://github.com/Homebrew/formula-patches

File: Formula/aurora-cli.rb (hunk 1 above)
Comment (thinker0) [reply Formula/aurora-cli.rb]: @ilovezfs Thanks

File: Formula/aurora-cli.rb (hunk 2 above)
Comment (thinker0) [reply Formula/aurora-cli.rb]: @ilovezfs Thanks 

https://github.com/Homebrew/formula-patches/pull/206

File: Formula/aurora-cli.rb [hunk 3]
```diff
@@ -11,9 +11,17 @@ class AuroraCli < Formula
     sha256 "62e565f0d65601f88b11d8c5f5bfad89d2f1661144072876fc705e50b4284329" => :yosemite
   end
 
+  if MacOS.version == :high_sierra
+    patch do
+      url "https://raw.githubusercontent.com/Homebrew/formula-patches/master/aurora-cli/pants_version_1.4.0_dev20.diff"
```
Comment (ilovezfs) [inline Formula/aurora-cli.rb]: https://raw.githubusercontent.com/Homebrew/formula-patches/21a7e7d/aurora-cli/pants_version_1.4.0_dev20.diff

**Comment by thinker0** [comment]
@ilovezfs ^______________^

**Comment by ilovezfs** [comment]
Thanks again @thinker0! Shipped.

**Comment by thinker0** [comment]
@ilovezfs Thanks ^^;

**Pull request closed without merging**